lib/elixir/lib/exception.ex

```diff
@@ -2509,7 +2509,7 @@ defmodule File.LinkError do
   def message(exception) do
     formatted = IO.iodata_to_binary(:file.format_error(exception.reason))
 
-    "could not #{exception.action} from #{inspect(exception.existing)} to " <>
+    "could not #{exception.action} to #{inspect(exception.existing)} from " <>
```

Comment:
To keep consistency with the error on line 2484, shall we keep it as from-to but swap the exception.existing with exception.new?

Reply:
Sounds good to me.

Reply:
(done btw)